src/AlgebraicGeometry/ToricVarieties/ToricDivisors/properties.jl

```diff
@@ -153,7 +153,10 @@ julia> is_very_ample(td)
 false
 ```
 """
-@attr Bool is_very_ample(td::ToricDivisor) = pm_object(td).VERY_AMPLE
+@attr Bool function is_very_ample(td::ToricDivisor)
+  @req is_complete(toric_variety(td)) "Very ampleness test is only implemented for complete toric varieties. ([Def 6.1.9 CLS11])"
+  pm_object(td).VERY_AMPLE
```

Comment:
```suggestion
  return pm_object(td).VERY_AMPLE
```
we usually add the explicit `return` in multiline functions

Reply:
```suggestion
  return pm_object(td).VERY_AMPLE::Bool
```
Polymake properties should usually get a type annotation (as it cannot be inferred).